javascript/public/manifest.json

```diff
@@ -1,25 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "",
+  "short_name": "",
   "icons": [
```

Comment:
Please consider changing:
* `"name": "",` to `"name": "UCSB Courses Search"
* `"short_name": "",` to `"short_name": "Courses Search",`

Reply:
Thanks for catching that. Just made the change!